base/grafana/grafana.Deployment.yaml

```diff
@@ -24,8 +24,21 @@ spec:
         app: grafana
     spec:
       containers:
-      - image: index.docker.io/sourcegraph/grafana:6.3.3@sha256:2f68b9b1542e7d75459d983b606d2fdd1c11a75610464e3d7a6ced4f3ac474bf
+      - image: index.docker.io/sourcegraph/grafana:6.3.3-1-internal-uwe
         terminationMessagePolicy: FallbackToLogsOnError
+        env:
+        - name: GF_AUTH_ANONYMOUS_ENABLED
+          value: "true"
+        - name: GF_AUTH_ANONYMOUS_ORG_NAME
+          value: "Main Org."
+        - name: GF_AUTH_ANONYMOUS_ORG_ROLE
+          value: "Editor"
+        - name: GF_USERS_ALLOW_SIGN_UP
+          value: "false"
+        - name: GF_USERS_AUTO_ASSIGN_ORG
+          value: "true"
+        - name: GF_USERS_AUTO_ASSIGN_ORG_ROLE
+          value: "Editor"
```

Comment:
It would be ideal to move these into the container itself now that we're using our own. i.e. in the Dockerfile using:

```
ENV GF_AUTH_ANONYMOUS_ENABLED true
...
```

The reason we prefer this is to make our YAML simpler, and it becomes easier to keep this repo and other repos deploying this container in sync.

Reply:
that's a good idea. if it's ok with you i'll do this in a subsequent PR because i have a couple of changes i like to make to the grafana image and i'll include this then. i'll clean up the envs afterwards